src/utils/common.ts

```diff
@@ -26,6 +27,11 @@ export const roundUpTime = (time: number, period: Period) => {
   return (Number(time / offset) * offset) + offset;
 }
 
+export function parseBigNumberToString(bn: BigNumber, decimals = 0): string {
+  if (!bn) return "-";
+  return parseFloat(ethers.utils.formatUnits(bn)).toFixed(decimals);
```

Comment:
you should reuse the function below if you like
```suggestion
  return convertStringToFloat(ethers.utils.formatUnits(bn), decimals);
```

Reply:
would also try and keep the naming convention

`convertStringToFloat` and `convertBigNumberToString`

Reply:
ty, will update